index.bs

```diff
@@ -10202,6 +10269,21 @@ the <code>typeof</code> operator will return "function" when applied to an inter
 </div>
 
 
+<h4 id="legacy-window-alias">Legacy window alias</h4>
```

Comment:
Done. Only changed "The property has attributes" to "The property has the attributes" for consistency with the surrounding paragraphs.